File: android/src/main/java/com/fidelreactlibrary/events/BridgeLibraryEvent.java
package com.fidelreactlibrary.events;

public enum BridgeLibraryEvent {
    RESULT_AVAILABLE("ResultAvailable"),

    private final String eventName;

    BridgeLibraryEvent(String eventName) {
        this.eventName = eventName;
    }

    public String getEventName() {
        return this.eventName;
    }
}

File: ios/Events/BridgeLibraryEvent.swift
//
//  BridgeLibraryEvent.swift
//  fidel-react-native
//
//  Created by Corneliu Chitanu on 07/06/23.
//

import Foundation

enum BridgeLibraryEvent: String, CaseIterable {
    case resultAvailable = "ResultAvailable"
    case cardVerificationStarted = "CardVerificationStarted"
    case cardVerificationChoiceSelected = "CardVerificationChoiceSelected"
}
